### ISSUE TITLE
Add winetricks

### ISSUE THREAD

**Issue by Asuka-pr8** [olegos2/mobox#109]
I think it will be easier for users to be able to use winetricks to get stuff like fonts or video codecs.

Thanks for your hard work on everything I really love the project so far.

**Comment by pikatwinky** [comment]
+1 to this. I think winetricks was very useful to install fonts when using other locales. In the meanwhile, im wondering if is there a way to do this (install fonts or packages) manually?

**Comment by Toni500github** [comment]
+1 Need winetricks for installing dotnet packages for some apps

**Comment by screeny05** [comment]
I got it working on my android.

First, you need some prerequisites:
```
export GLIBC_BIN=$PREFIX/glibc/bin
pkg install cabextract unzip p7zip which

# download bash for x86
wget https://github.com/ptitSeb/box64/raw/main/tests/bash
mv bash $GLIBC_BIN/bash86
chmod +x $GLIBC_BIN/bash86

# download winetricks
wget https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
mv winetricks $GLIBC_BIN/winetricks
chmod +x $GLIBC_BIN/winetricks
```

When you have all this, you can run winetricks like so:
```
unset LD_PRELOAD
WINESERVER=$PREFIX/glibc/bin/wineserver WINE=$PREFIX/glibc/bin/wine64 $PREFIX/glibc/bin/box64 $PREFIX/glibc/bin/bash86 $PREFIX/glibc/bin/winetricks
export LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so
```

**Comment by Toni500github** [comment]
> I got it working on my android.
> 
> First, you need some prerequisites:
> ```
> export GLIBC_BIN=$PREFIX/glibc/bin
> pkg install cabextract unzip p7zip which
> 
> # download bash for x86
> wget https://github.com/ptitSeb/box64/raw/main/tests/bash
> mv bash $GLIBC_BIN/bash86
> chmod +x $GLIBC_BIN/bash86
> 
> # download winetricks
> wget https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
> mv winetricks $GLIBC_BIN/winetricks
> chmod +x $GLIBC_BIN/winetricks
> ```
> 
> When you have all this, you can run winetricks like so:
> ```
> LD_PRELOAD= WINESERVER=$PREFIX/glibc/bin/wineserver WINE=$PREFIX/glibc/bin/wine64 $PREFIX/glibc/bin/box64 $PREFIX/glibc/bin/bash86 $PREFIX/glibc/bin/winetricks
> ```

Thanks it works, but only if I run with --help arg. Is it supposed to be so slow at first or?
● ● ● LD_PRELOAD= WINESERVER=$PREFIX/glibc/bin/wineserver WINE=$PREFIX/glibc/bin/wine64 $PREFIX/glibc/bin/box64 $PREFIX/glibc/bin/bash86 $PREFIX/glibc/bin/winetricks
Dynarec for ARM64, with extension: ASIMD AES CRC32 PMULL PageSize:4096 Running on Kryo-V2 Cortex-A73 with 8 Cores
Params database has 40 entries
Box64 with Dynarec v0.2.5  built on Nov 10 2023 14:30:39
Counted 43 Env var
BOX64 LIB PATH: ./:lib/:lib64/:x86_64/:bin64/:libs64/:/data/data/com.termux/files/usr/glibc/lib/x86_64-linux-gnu/
BOX64 BIN PATH: ./:bin/:/data/data/com.termux/files/usr/bin/:/data/data/com.termux/files/home/.local/bin/:/data/data/com.termux/files/usr/bin/
Looking for /data/data/com.termux/files/usr/glibc/bin/bash86
argv[1]="/data/data/com.termux/files/usr/glibc/bin/winetricks"
Rename process to "bash86"
Using native(wrapped) libtinfo.so.6
Using native(wrapped) libdl.so.2
Using native(wrapped) libc.so.6
Using native(wrapped) ld-linux-x86-64.so.2
Using native(wrapped) libpthread.so.0
Using native(wrapped) libutil.so.1
Using native(wrapped) librt.so.1

**Comment by screeny05** [comment]
Yeah, it's not the fastest. But installing directplay worked for me.

**Comment by Toni500github** [comment]
@screeny05 I end up to get this error

`wine: could not load kernel32.dll, status c0000135`

**Comment by screeny05** [comment]
I'm sorry, i'm no wine-magician. Maybe someone more experienced than me can help here?

**Comment by sdtthdsr7u** [comment]
> I got it working on my android.
> 
> First, you need some prerequisites:
> 
> ```
> export GLIBC_BIN=$PREFIX/glibc/bin
> pkg install cabextract unzip p7zip which
> 
> # download bash for x86
> wget https://github.com/ptitSeb/box64/raw/main/tests/bash
> mv bash $GLIBC_BIN/bash86
> chmod +x $GLIBC_BIN/bash86
> 
> # download winetricks
> wget https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
> mv winetricks $GLIBC_BIN/winetricks
> chmod +x $GLIBC_BIN/winetricks
> ```
> 
> When you have all this, you can run winetricks like so:
> 
> ```
> LD_PRELOAD= WINESERVER=$PREFIX/glibc/bin/wineserver WINE=$PREFIX/glibc/bin/wine64 $PREFIX/glibc/bin/box64 $PREFIX/glibc/bin/bash86 $PREFIX/glibc/bin/winetricks
> ```

so what should i do to install dll，i am sorry to trouble you but i tried it for several days.
@screeny05

**Comment by NyaMisty** [comment]
As of 20241202, this modded script worked (for wow64 version): 
```
export GLIBC_BIN=$PREFIX/glibc/bin
pkg install cabextract unzip p7zip which

# download bash for x86
wget https://github.com/ptitSeb/box64/raw/main/tests/box64-bash
mv bash $GLIBC_BIN/box64-bash
chmod +x $GLIBC_BIN/box64-bash

# download winetricks
wget https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
mv winetricks $GLIBC_BIN/winetricks
chmod +x $GLIBC_BIN/winetricks
```

```
LD_PRELOAD= WINESERVER=$PREFIX/glibc/bin/wineserver WINE=$PREFIX/glibc/bin/wine64 $PREFIX/glibc/bin/box64 $PREFIX/glibc/bin/box64-bash $PREFIX/glibc/bin/winetricks
```

Note because most winetricks needs GUI, so you may also need to execute `export DISPLAY=:0` beforehand
